src/rules/mod.rs

```diff
@@ -259,7 +260,10 @@ impl<'i> CssRuleList<'i> {
     for mut rule in self.0.drain(..) {
       match &mut rule {
         CssRule::Keyframes(keyframes) => {
-          if context.unused_symbols.contains(keyframes.name.0.as_ref()) {
+          if context
+            .unused_symbols
+            .contains(&keyframes.name.to_css_string(Default::default()).unwrap())
```

Comment:
This will result in the name being quoted/escaped, but the `unused_symbols` list won't be. You'll need to do a `match` on the keyframe name to pull out the embedded string.